.github/ci/unbottled_dependencies.bash

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+# Copyright (C) 2024 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# The script will print the unbottled dependencies from the osrf/simulation
+# tap of a specified formula.
+#
+# Usage:
+# $ ./unbottled_dependencies.bash <formula>
+#
+# For example, to print the unbottled dependencies of gz-harmonic from the
+# osrf/simulation tap:
+#
+# ./unbottled_dependencies.bash gz-harmonic
+
+FORMULA=${1}
+
+if [[ $# -ne 1 ]]; then
+  echo "./unbottled_dependencies.bash <formula>"
+  exit 1
+fi
+
+for f in $(brew deps ${FORMULA} --full-name | grep osrf/simulation)
```

Comment:
Is `osrf/simulation` always at the beginning or end of the full name? You could either use `grep -F` to do a literal search for this, although it would match `gosrf/simulationz` as well. Or use an anchored expression such as `^osrf/simulation` for matching the start of the line.

Reply:
tightened up as `^osrf/simulation/` in https://github.com/osrf/homebrew-simulation/pull/2610/commits/d45f9e64382fc869d5834aae39c621d193087d6c